feature-libs/product-configurator/rulebased/components/config/default-configurator-ui-settings.config.ts

```diff
@@ -14,6 +14,6 @@ export const defaultConfiguratorUISettingsConfig: ConfiguratorUISettingsConfig =
         input: 500,
       },
       addRetractOption: false,
-      enableNavigationToConflict: false,
+      enableNavigationToConflict: true,
```

Comment:
@Larisa-Staroverova Please revert this change

Reply:
Done!